### ISSUE TITLE
Multimodal support with Gemma 4 model

### ISSUE THREAD

**Issue by sshonTT** [tenstorrent/tt-xla#4702]
The current version of Gemma 4 only supports text. However, since it has multimodal capabilities, we should be able to run it with image, audio, and video inputs.

**Comment by sshonTT** [comment]
I have a working branch for image : https://github.com/tenstorrent/tt-xla/tree/sshon/multimodal-gemma4

**Comment by sshonTT** [comment]
logits Softcap is required for audio
